app/backend/api/posts.py

```diff
@@ -66,3 +66,10 @@ def justin():
     return jsonify({
         "Suck": "Butt"
     })
+
+@posts.route('/factorial/<num>')
+def facotrial(num):
+
+    return jsonify({
```

Comment:
This doesn't implement it, just returns the number

Reply:
This  is done.